styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -157,6 +159,7 @@
   private final EventConsumerFactory eventConsumerFactory;
   private final WorkflowConsumerFactory workflowConsumerFactory;
   private final WorkflowExecutionGateFactory executionGateFactory;
+  private final HouseCleaningOperationFactory housecleaningOperationFactory;
```

Comment:
Nitpick: if it's HouseCleaningFoo then the lower camel case is more consistently houseCleaningFoo...

Reply:
Oh yeah, my bad, should just call it `HousecleaningOperationFacotry`.